src/tenant/tests/test_admin.py

```diff
@@ -508,6 +521,11 @@ def setUp(self):
 
         self.client = TenantClient(self.public_tenant)
 
+        # We need to check if library tenant exist
+        # Since library is created elsewhere it will fail only if full tests are ran
+        # See Issue #1590 for more details
```

Comment:
Including issue #'s as reference are awesome, put put the entire link.

Reply:
done